dev/check-license

```diff
@@ -43,6 +43,7 @@ acquire_rat_jar () {
     unzip -tq $JAR &> /dev/null
     if [ $? -ne 0 ]; then
       # We failed to download
+      rm $JAR
```

Comment:
Actually I also just noticed that this should be quoted as `"$JAR"`, and above in the `unzip` command, to take care with paths containing spaces.